### PR TITLE
Add liquibase(create tables and insert) and fix bug with docker and l…

### PR DESCRIPTION
**Pull request by vov4ik-v**
…iquibase

### CODE REVIEW

File: src/main/java/com/spm/vasylyshyn/CounterDisplayServerApplication.java [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class CounterDisplayServerApplication {
 
 
     public static void main(String[] args) {
-        System.out.println("Hello 2");
```
Comment (BogdanDidukh2003) [inline src/main/java/com/spm/vasylyshyn/CounterDisplayServerApplication.java:21]: Is this relevant?

File: Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ WORKDIR /app
 COPY pom.xml .
 COPY src ./src
 # Build the application using Maven
-RUN mvn clean package -DskipTests
+RUN mvn install -DskipTests
```
Comment (BogdanDidukh2003) [inline Dockerfile:14]:
```suggestion
RUN mvn install -D skipTests
```
Isn't this a typo?

File: src/main/resources/db/changelog/scripts/create_device_table.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+(
+    number_of_device BIGINT AUTO_INCREMENT PRIMARY KEY,
+    cantora_name    VARCHAR(255) default null,
+    counter_type    INT  default null,
+    address        VARCHAR(255)  default null,
+    frequency      BIGINT  default null,
+    password       VARCHAR(255)  default null,
+    price          INT  default null,
+    user_id        BIGINT  default null,
+    created_date    VARCHAR(255) NOT NULL,
```
Comment (BogdanDidukh2003) [inline src/main/resources/db/changelog/scripts/create_device_table.sql]: Fix spacing (1 space between words)

**Review by BogdanDidukh2003 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (vov4ik-v) [reply Dockerfile:14]: No, this isn`t a typo

File: src/main/java/com/spm/vasylyshyn/CounterDisplayServerApplication.java (hunk 1 above)
Comment (vov4ik-v) [reply src/main/java/com/spm/vasylyshyn/CounterDisplayServerApplication.java:21]: No. i`ll delete it

File: src/main/resources/db/changelog/scripts/create_device_table.sql (hunk 3 above)
Comment (vov4ik-v) [reply src/main/resources/db/changelog/scripts/create_device_table.sql]: Fix it

**Pull request merged**